tests/main/try-with-hooks/task.yaml

```diff
@@ -1,7 +1,7 @@
 summary: Reproduce a known issue of snap try failure with hooks
 
 prepare: |
-  cp -R "$TESTSLIB"/snaps/basic ./
+  cp -a "$TESTSLIB"/snaps/basic ./
```

Comment:
+1 on the change but curious on the motivation that lead to it

Reply:
Default umask got changed to `0077`. The permissions are already set properly in our tree, but without `-a` they'd get reset to defaults. Then snap verification would bail because the directories are not world readable and so on.